test/unsupported-httpversion.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const net = require('net')
+const t = require('tap')
+const Fastify = require('../fastify')
+
+t.test('Will return 505 HTTP error if HTTP version (default) is not supported', t => {
```

Comment:
Can we also add tests that cover not just the `505 HTTP Version Not Supported`, but also a case/cases where the server responds with `2xx` status code? We want to make sure the server is able to respond with something else than status code `505`.

Reply:
I think they would be overkill. There are plenty of those in the other tests :)

Reply:
The existing test cases already handle it. It is not necessary to add.

Reply:
Ok cool, I wasn't aware of what the other existing tests cover.